patches/tinyemu/tinyemu/Makefile

```diff
@@ -67,7 +67,7 @@ EMU_OBJS:=virtio.o pci.o fs.o cutils.o iomem.o simplefb.o \
 
 ifdef CONFIG_SLIRP
 CFLAGS+=-DCONFIG_SLIRP
-EMU_OBJS+=$(addprefix slirp/, bootp.o ip_icmp.o mbuf.o slirp.o tcp_output.o cksum.o ip_input.o misc.o socket.o tcp_subr.o udp.o if.o ip_output.o sbuf.o tcp_input.o tcp_timer.o)
+EMU_OBJS+=$(addprefix slirp/, bootp.o ip_icmp.o mbuf.o slirp.o tcp_output.o cksum.o ip_input.o misc.o socket.o tcp_subr.o udp.o if.o ip_output.o sbuf.o tcp_input.o tcp_timer.o wasi_socket_ext.o)
```

Comment:
As of now, it's fine that the guest VM and the container share the network namespace to make the network interface visible to the container. You can change the runc spec to do so:

https://github.com/ktock/container2wasm/blob/bc5b06809d99e052dc41ae4396fb3cdffa85c3f0/cmd/create-spec/main.go#L247

```diff
-       s, err := ctdoci.GenerateSpecWithPlatform(ctdCtx, nil, "linux/riscv64", &ctdcontainers.Container{})
+       s, err := ctdoci.GenerateSpecWithPlatform(ctdCtx, nil, "linux/riscv64", &ctdcontainers.Container{},
+               ctdoci.WithHostNamespace(specs.NetworkNamespace),
+       )
```

Reply:
Handled.